test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -24,26 +24,65 @@ test('IndexTable - bulk action', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
   await indexTablePage.goto();
 
+  let isCheckboxCorrect: boolean, tempText: string | null;
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues(
+    new Array(5).fill(false)
+  );
+  expect(isCheckboxCorrect).toBe(true);
+
   await indexTablePage.checkboxes.first().click();
   await expect(indexTablePage.bulkAction).toBeVisible();
-  await expect(indexTablePage.canvas).toHaveScreenshot(
-    'indextable-bulk-action-selected-all.png'
+
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues(
+    new Array(5).fill(true)
   );
+  expect(isCheckboxCorrect).toBe(true);
+
+  tempText = await indexTablePage.tableHeadRow.textContent();
+  expect(tempText).toBe('4 selected candidates');
+
+  tempText = await indexTablePage.bulkAction.textContent();
+  expect(tempText).toContain('Reject');
+  expect(tempText).toContain('Move to');
 
   await indexTablePage.checkboxes.last().click();
 
   await expect(indexTablePage.bulkAction).toBeVisible();
 
-  await expect(indexTablePage.canvas).toHaveScreenshot(
-    'indextable-bulk-action-selected-row.png'
-  );
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues([
+    false,
+    true,
+    true,
+    true,
+    true,
```

Comment:
i'm expecting the last checkbox should have been false as well. since previously this was checked, clicking it should've updated the checked state to false

Reply:
can you double check this as well it seems the assertion is false positive and could affect the other assertions using the same function. the `selectAllTableHeadText` assertion below is correct which is why this seems to be false positive

Reply:
> i'm expecting the last checkbox should have been false as well. since previously this was checked, clicking it should've updated the checked state to false

Yes, the last checkbox is `false`, which is on line 53. I also thought that the order should be based on the DOM, but not sure why it turns out that the order does not follow the DOM (pretty sure it is reversed there).
![image](https://github.com/glints-dev/glints-aries/assets/67429929/b691cb87-3a27-4444-b1d7-26896c082e7e)
In that picture, the very first checkbox (which shows 3 selected candidates) is `checked` (so it should be `true` as indicated on line 57), the difference is only on the input-id (so there is a minus sign there instead of tick), maybe I could also have a test for that hmm.

Let me try to check for a moment and update you again.

Reply:
checked and fixed